modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring route admission policy
+
+By default, {microshift-short} prevents routes in multiple namespaces from using the same hostname. To accommodate different use cases, you can use the configuration option below to disable the `namespaceOwnership` check. This allows routes to claim different paths of the same hostname across namespaces.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `InterNamespaceAllowed` as shown in the following example to disable the check:
```

Comment:
```suggestion
.  To prevent routes in different namespaces from claiming the same hostname, update the `namespaceOwnership` field value to `Strict` in the {microshift-short} `config.yaml` file as shown in the following example:
```
* The _parameter_ is the "key" part of the YAML, the thing left of the colon, so in this case, `ingress.routeAdmissionPolicy.namespaceOwnership` (or just `namespaceOwnership` since we do give the stanza here). The _value_ of the field the parameter defines is what we are changing.

* We want to keep "in the following example" as the last thing before the example and move the clause earlier in the sentence. It's more user-focused and is important for accessibility (think screen readers). 

* It is worthwhile to ask @ShudiLi what happens if a user does delete the whole `routeAdmissionPolicy.namespaceOwnership` parameter. We have some funky behavior with other parameters based on whether they are deleted or left empty.

Reply:
updated suggestions. 
Checking with @ShudiLi

Reply:
@ShaunaDiaz If the whole `routeAdmissionPolicy.namespaceOwnership` parameter is delete, the default value `InterNamespaceAllowed` will be set to `routeAdmissionPolicy.namespaceOwnership`. If there isn't the config.yaml file, all parameters are set to the default values(If there is a parameter in the config.yaml file, the default value will be set to it)